.github/workflows/build.yml

```diff
@@ -43,8 +43,40 @@ jobs:
         run: rustup target add ${{ matrix.target }}
       - name: Build
         run: cargo build --verbose --release --target ${{ matrix.target }}
+      - if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+        name: Downloading dummy service for packaging
+        uses: actions/download-artifact@v4
+        with:
+          name: dummy-GalaxyCommunication.exe
+          path: $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists
+      - if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+        name: Unpacking dummy service
+        run: | 
+          unzip -j $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/dummy-GalaxyCommunications.exe.zip -d $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists
+          rm $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/dummy-GalaxyCommunications.exe.zip
+          rm $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/GalaxyCommunications_here
+        
+      - if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+        name: Add Steam Deck shortcut & service files
+        run: | 
+          cp $GITHUB_WORKSPACE/steamdeck/comet_shortcut.sh target/${{ matrix.target }}/release/comet_shortcut.sh
+          cp $GITHUB_WORKSPACE/steamdeck/USAGE.md target/${{ matrix.target }}/release/USAGE.md
+          cp -R $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx target/${{ matrix.target }}/release/pfx
+          cp $GITHUB_WORKSPACE/dummy-service/comet-dummy-service.bat target/${{ matrix.target }}/release/comet-dummy-service.bat
```

Comment:
I think there is no need for this copying to be here, we can reference those files directly during upload

Reply:
Done.